flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/FunctionCatalog.java

```diff
@@ -116,14 +125,60 @@ public void registerScalarFunction(String name, ScalarFunction function) {
 	}
 
 	public String[] getUserDefinedFunctions() {
```

Comment:
We probably also need to take care of namespace (catalog.database) of functions.